dexbot/strategies/external_feeds/ccxt_feed.py

```diff
@@ -1,123 +1,106 @@
     except ccxt.DDoSProtection as e:
         print(type(e).__name__, e.args, 'DDoS Protection (ignoring)')
     except ccxt.RequestTimeout as e:
-        print(type(e).__name__, e.args, 'Request Timeout (ignoring)')
+        print(type(e).__name__, e.args,
+            'Request Timeout (ignoring)')
     except ccxt.ExchangeNotAvailable as e:
-        print(type(e).__name__, e.args, 'Exchange Not Available due to downtime or maintenance (ignoring)')
+        print(type(e).__name__, e.args,
+            'Exchange Not Available due to downtime or maintenance (ignoring)')
     except ccxt.AuthenticationError as e:
-        print(type(e).__name__, e.args, 'Authentication Error (missing API keys, ignoring)')
-        
+        print(type(e).__name__, e.args,
+            'Authentication Error (missing API keys, ignoring)')
     return ticker
 
 
-
 async def fetch_ticker(exchange, symbol):
     ticker = await exchange.fetchTicker(symbol)
-    print(exchange.id, symbol, ticker)
+    await exchange.close()
     return ticker
 
-
-async def fetch_tickers(exchange):
-    await exchange.load_markets()
-    print(exchange.id, 'fetching all tickers by simultaneous multiple concurrent requests')
-    symbols_to_load = get_active_symbols(exchange)
-    input_coroutines = [fetch_ticker(exchange, symbol) for symbol in symbols_to_load]
-    tickers = await asyncio.gather(*input_coroutines, return_exceptions=True)
-    for ticker, symbol in zip(tickers, symbols_to_load):
-        if not isinstance(ticker, dict):
-            print(exchange.id, symbol, 'error')
-        else:
-            print(exchange.id, symbol, 'ok')
-    print(exchange.id, 'fetched', len(list(tickers)), 'tickers')
-
-
 async def print_ticker(symbol, id):
     # verbose mode will show the order of execution to verify concurrency
     exchange = getattr(accxt, id)({'verbose': True})
     print(await exchange.fetch_ticker(symbol))
+    await exchange.close()
 
 
-###### unit tests ######
+# unit tests
 @click.group()
 def main():
     pass
 
 
 @main.command()
-def test_async():
-
+def test_async2():
+    """
+    get all tickers from multiple exchanges using async
+    """
     symbol = 'ETH/BTC'
     print_ethbtc_ticker = functools.partial(print_ticker, symbol)
     [asyncio.ensure_future(print_ethbtc_ticker(id)) for id in [
-            'bitfinex',
-            'binance',
-            'kraken',
-            'gdax',
-            'bittrex',
-            ]]
+        'bitfinex',
+        'binance',
+        'kraken',
+        'gdax',
+        'bittrex',
+    ]]
     pending = asyncio.Task.all_tasks()
     loop = asyncio.get_event_loop()
     loop.run_until_complete(asyncio.gather(*pending))
 
-#    asyncio.get_event_loop().run_until_complete(fetch_tickers(accxt.bitfinex({
-#                    'enableRateLimit': True,  # this option enables the built-in rate limiter
-#                    })))
 
+@main.command()
+def test_async():
+    """
+    get ticker for bitfinex using async
+    """
```

Comment:
` """ Get ticker for bitfinex using async
  """` Would follow the rest of the project better. Using correct quotes now which is good :+1: